<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>RotaryMotorPropDialog</class>
 <widget class="QDialog" name="RotaryMotorPropDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Rotary Motor Properties</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="nameTextLabel" >
           <property name="text" >
            <string>Name :</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabel1" >
           <property name="font" >
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text" >
            <string>Torque :</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy" >
            <cstring>lineEdit9</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabel1_2_3" >
           <property name="text" >
            <string>Initial Speed :</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy" >
            <cstring>lineEdit9_2</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabel1_2" >
           <property name="text" >
            <string>Final Speed :</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy" >
            <cstring>lineEdit9_2</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" colspan="2" >
          <widget class="QLineEdit" name="nameLineEdit" >
           <property name="toolTip" >
            <string>Name of rotary motor.</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading</set>
           </property>
           <property name="readOnly" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLineEdit" name="torqueLineEdit" >
           <property name="toolTip" >
            <string>Simulations will begin with the motor's torque set to this value.</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLineEdit" name="speedLineEdit" >
           <property name="toolTip" >
            <string>The final velocity of the motor's flywheel.</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLabel" name="textLabel1_4" >
           <property name="text" >
            <string>nN-nm</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QLabel" name="textLabel2" >
           <property name="text" >
            <string>GHz</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLabel" name="textLabel2_2" >
           <property name="text" >
            <string>GHz</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLineEdit" name="initialSpeedLineEdit" >
           <property name="toolTip" >
            <string>The beginning velocity of the motor's flywheel.</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="dampers_textlabel" >
         <property name="toolTip" >
          <string>If checked, dampers are enabled in the simulator.</string>
         </property>
         <property name="text" >
          <string>Dampers :</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_5" >
         <property name="toolTip" >
          <string>If checked, the torque value is applied to the motor's atoms during minimization.</string>
         </property>
         <property name="text" >
          <string>Enable in Minimize (experimental) :</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="dampers_checkbox" >
         <property name="toolTip" >
          <string>If checked, dampers are enabled in the simulator.</string>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="enable_minimize_checkbox" >
         <property name="toolTip" >
          <string>If checked, the torque value is applied to the motor's atoms during minimization.</string>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>Size and Color</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="3" column="1" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="jig_color_pixmap" >
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="scaledContents" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="choose_color_btn" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Choose...</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>46</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="textLabel1_2_2_2" >
        <property name="text" >
         <string>Spoke Radius :</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel1_3" >
        <property name="text" >
         <string>Motor Length :</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel1_2_2" >
        <property name="text" >
         <string>Motor Radius :</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="colorTextLabel" >
        <property name="text" >
         <string>Color :</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="lengthLineEdit" >
        <property name="alignment" >
         <set>Qt::AlignLeading</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="sradiusLineEdit" >
        <property name="alignment" >
         <set>Qt::AlignLeading</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="radiusLineEdit" >
        <property name="alignment" >
         <set>Qt::AlignLeading</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="textLabel3_2" >
        <property name="text" >
         <string>Angstroms</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="textLabel3" >
        <property name="text" >
         <string>Angstroms</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="textLabel3_3" >
        <property name="text" >
         <string>Angstroms</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_btn" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>torqueLineEdit</tabstop>
  <tabstop>initialSpeedLineEdit</tabstop>
  <tabstop>speedLineEdit</tabstop>
  <tabstop>enable_minimize_checkbox</tabstop>
  <tabstop>lengthLineEdit</tabstop>
  <tabstop>radiusLineEdit</tabstop>
  <tabstop>sradiusLineEdit</tabstop>
  <tabstop>choose_color_btn</tabstop>
  <tabstop>ok_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>RotaryMotorPropDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok_btn</sender>
   <signal>clicked()</signal>
   <receiver>RotaryMotorPropDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
